google_analytics/regressors.py

```diff
@@ -1,13 +1,32 @@
 import multiprocessing
 from sklearn.linear_model import Ridge, Lasso
 from sklearn.ensemble import RandomForestRegressor
+import xgboost as xgb
 # try:
 #     from xgboost import XGBClassifier
 # except ImportError:
 #     print("XGBoost not imported.")
 from predictor import Predictor
 
 
+class XGboostPredictor(Predictor):
+    """
+    More information about this regressor at:
+    https://xgboost.readthedocs.io/en/latest/python/python_api.html
+    """
+    name = 'XGboost Predictor'
+
+    def __init__(self, max_depth=3, learning_rate=0.1, n_estimators=100, silent=True, objective='reg:linear',
+                 booster='gbtree', n_jobs=1, nthread=None, gamma=0, min_child_weight=1, max_delta_step=0,
+                 subsample=1, colsample_bytree=1, colsample_bylevel=1, reg_alpha=0, reg_lambda=1, scale_pos_weight=1,
+                 base_score=0.5, random_state=0, seed=None, missing=None, name=name):
+        super().__init__(name)
```

Comment:
The `name`  attribute is redudant. You can do the same by using: ` super().__init__()`